docusaurus/docs/reactnative/additional_resources.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: additional-resources
+sidebar_position: 8
+title: Additional Resources
+---
+
+Here's a list of resource links that could be helpful during development while using the Stream Chat React Native SDK library.
+
+## Internal Resources
+
+### [Stream Chat React Native GitHub](https://github.com/GetStream/stream-chat-react-native)
+
+### [Stream Chat React SDK Native - Demos](https://github.com/GetStream/react-native-samples)
+
+### [Stream Chat React Native SDK Tutorial](https://getstream.io/chat/react-native-chat/tutorial/)
+
+### [Stream JS Client](https://getstream.io/chat/docs/javascript/?language=javascript)
+
+### [Stream Chat API tutorial](https://getstream.io/chat/get_started/)
+
+## External Resources
+
+### [Typescript - Generics](https://www.typescriptlang.org/docs/handbook/2/generics.html)
+
+Our SDK is written in Typescript and heavily uses generics in order to support all our customizations.
+
+### [React - Getting Started](https://reactjs.org/docs/getting-started.html)
+
+### [React - Context](https://reactjs.org/docs/context.html)
+
+We heavily use contexts in our SDK to pass down custom components and properties.
+
+### [React - Hooks](https://reactjs.org/docs/hooks-intro.html)
+
+Starting on 3.0 we moved our logics to hooks everywhere in the SDK, so take a look at the docs.
```

Comment:
no need to mention 3.0, these docs are all fresh with no 2.0 references, had this same change on some of mine when writing.